app/views/agent/partials/RegistrationPartial.scala.html

```diff
@@ -51,41 +51,39 @@
 }
 
 @cancelRegistration = {
-    <div id="cancel-vat" class="column-one-third">
-        <h3 class="heading-small">
-            @cancelVatLink
-        </h3>
-        <p>@messages(cancelVatContentKey)</p>
+    <div class="govuk-grid-column-one-third" id="cancel-vat">
+        <h3 class="govuk-heading-s">@cancelVatLink</h3>
+        <p class="govuk-body">@messages(cancelVatContentKey)</p>
     </div>
 }
 
 @historicDeregistration(date: LocalDate) = {
-    <div id="cancel-vat" class="column-one-third">
-        <h3 class="heading-small">
-            @messages("registrationPartial.historicDeregistration.title")
-        </h3>
-        <p>@messages("registrationPartial.historicDeregistration.content", displayDate(date))</p>
-        <p><a id="agent-services-link" href="@appConfig.onlineAgentServicesUrl" target="_blank">@messages("registrationPartial.historicDeregistration.link")</a></p>
+    <div class="govuk-grid-column-one-third" id="cancel-vat">
+        <h3 class="govuk-heading-s">@messages("registrationPartial.historicDeregistration.title")</h3>
+        <p class="govuk-body">@messages("registrationPartial.historicDeregistration.content", displayDate(date))</p>
+        <p class="govuk-body">
+            <a id="agent-services-link" href="@appConfig.onlineAgentServicesUrl" target="_blank" class="govuk-link">
```

Comment:
For all links that have `target="_blank"` can you also add `rel="noreferrer noopener"`